src/Aquifer.API/Endpoints/Resources/Content/Helpers.cs

```diff
@@ -68,20 +73,36 @@ public static async Task CreateNewDraft(AquiferDbContext dbContext,
         {
             await historyService.AddAssignedUserHistoryAsync(newResourceContentVersion, assignedUserId, user.Id, ct);
         }
-        else
-        {
-            await historyService.AddSnapshotHistoryAsync(newResourceContentVersion, user.Id, ResourceContentStatus.New, ct);
-        }
 
         var resourceContent = await dbContext.ResourceContents.FindAsync([contentId], ct) ?? throw new ArgumentNullException();
```

Comment:
We should be using `First` or `Single`. .NET 9 introduced significant performance increases to LINQ, and it's much faster than `Find` now (it may not make much difference when constructing a query to the database, but for consistency, I would argue we shouldn't ever use `Find` now).

Reply:
From what I can tell it looks like `Find` is actually on `DbSet` and doesn't exist on Enumerables in LINQ. Seems nice to keep using for simple primary key lookups so I don't see much downside unless I'm missing something.